nodejs/src/utils.ts

```diff
@@ -1,21 +1,29 @@
 import Path from 'path';
 import FS from 'node:fs/promises';
-import { resolveKapetaVariables } from './variable-resolver';
-import { writeConfigTemplates } from './config-resolver';
-import { spawn } from '@kapeta/nodejs-process';
-import { writeDotEnvFile } from './dotenv-interpolation';
+import {resolveKapetaVariables, Variables, VariableType} from './variable-resolver';
+import {writeConfigTemplates} from './config-resolver';
+import {spawn} from '@kapeta/nodejs-process';
+import {DOTENV_FILE, writeDotEnvFile, writeEnvConfigFile} from './dotenv-interpolation';
 
 import YAML from 'yaml';
-import { Attachment, Kind } from '@kapeta/schemas';
-import { getAttachment, readAttachmentContent } from './attachments';
+import {Attachment, Kind} from '@kapeta/schemas';
+import {getAttachment, readAttachmentContent} from './attachments';
 import zlib from 'node:zlib';
 import util from 'node:util';
+import * as os from "os";
+import Config from "@kapeta/sdk-config";
 
 const gzip = util.promisify(zlib.gzip);
 const gunzip = util.promisify(zlib.gunzip);
 
+export const MAX_ENV_LENGTH = 256;
+export const KAPETA_CONFIG_ENV_VAR = 'KAPETA_CONFIG_PATH';
 export const KAPETA_YML = 'kapeta.yml';
 
+// We read environment variables from a json file to avoid argument length limits since some of
+// these might be quite long
+export const KAPETA_ENV_CONFIG_FILE = 'kapeta.env.config.json';
```

Comment:
I think it might be better with a env style file instead of json.
Like:
Key=value
I think that's the style most language and libraries would expect an env file to be in.

Reply:
I was thinking that at first as well - but realized that most languages do not actually have great support for reading such a format - all languages we currently support have very good support for reading JSON - which is why I chose that. (This is an internally generated and read file format - the simpler the better imo)